priv/repo/seeds.exs

```diff
@@ -1287,6 +1294,7 @@ uren_params = %{
   base_speed: 0.68,
   base_size: 100.0,
   base_health: 400,
+  base_attack: 1,
```

Comment:
Why do the characters need this parameter at all if its always 1?

Reply:
now that you mention it we don't need this right now. I was adding the field so that tanks would have a smaller base attack and damage dealers would have a higher base attack but left it as 1 to be the same as main and avoid having to do balance changes.